docs/hub/endpoints.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Hugging Face Hub Endpoints
+---
+
+# Hugging Face Hub API Endpoints
+
+
+We have open endpoints that you can use to retrieve information from the Hub as well as perform certain actions such as creating model repos. We offer a wrapper Python library, [`huggingface_hub`](https://github.com/huggingface/huggingface_hub), that allows easy access to these endpoints. We also provide a webhook to receive real-time incremental info about models. Enjoy!
+
+
+## Endpoints table
+
+| Endpoint                                                                         | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | `huggingface_hub` `HfApi` method       | Payload                                                                                                                                  |
+| -------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| /api/models  <br/>  GET                                                          | Get information from all models in the Hub. You can specify additional parameters to have more specific results.  <br/>- filter: Filter based on tags, such as `text-classification` or `spacy`.<br/>- sort: Property to use when sorting. <br/>- direction: Direction in which to sort. <br/>- limit: Limit the number of models fetched. <br/>- full: Whether to fetch most model data, such as all tags, the files, etc. <br/>- config: Whether to also fetch the repo config. | `list_models()`                        | ```params= {   filter:filter, full: full,    sort: sort,   direction: direction,   limit: limit,   config: config }```                   |
```

Comment:
(nit) how about putting parameters in backticks to imply that it's code? 
```suggestion
| /api/models  <br/>  GET                                                          | Get information from all models in the Hub. You can specify additional parameters to have more specific results.  <br/>- `filter`: Filter based on tags, such as `text-classification` or `spacy`.<br/>- `sort`: Property to use when sorting. <br/>- `direction`: Direction in which to sort. <br/>- `limit`: Limit the number of models fetched. <br/>- `full`: Whether to fetch most model data, such as all tags, the files, etc. <br/>- `config`: Whether to also fetch the repo config. | `list_models()`                        | ```params= {   filter:filter, full: full,    sort: sort,   direction: direction,   limit: limit,   config: config }```                   |
```